src/udgl/data-graphics/utils/color-maps.js

```diff
@@ -20,7 +20,7 @@ export function createCatColorMap(options) {
   return function catColorMap(v) {
     const i = getID(v);
     if (i >= 10 || i === undefined) {
-      return 'var(--cool-gray-200)';
+      return 'var(--cool-gray-400)';
```

Comment:
```suggestion
      return 'var(--cool-gray-300)';
```

Reply:
I'm a little concerned that this isn't enough contrast, but this is fine.